feg/gateway/services/n7_n40_proxy/n7/model_conversion.go

```diff
@@ -287,6 +343,44 @@ func GetPolicyReauthRequestProto(
 	}
 }
 
+func GetUsageMonitoringResponsesProto(
+	updateCtx *SmPolicyUpdateReqCtx,
+	smPolicyDecision *n7_sbi.SmPolicyDecision,
+) []*protos.UsageMonitoringUpdateResponse {
+	staticRules, dynamicRules, rulesToRemove := getPccRules(smPolicyDecision)
+	responses := getUsageMonitorsProtoForUpdate(updateCtx, smPolicyDecision)
+	if len(responses) == 0 {
+		responses = append(responses, NewUsageMonitoringUpdateResponseProto(updateCtx, true))
+	}
```

Comment:
why do you need this `if` here?

And if you need it, wouldint it be better to have it inside `getUsageMonitorsProtoForUpdate`? Or is this creating a fake `response` in case there is no response? Is it possible we discuss something similar for omnipresent rules?

Reply:
the reason  I don't like this is because we will forget in the future. Maybe we need to do this somewhere else (for example at the omnirules, this way we just have to do it once

Reply:
thanks. I now remember that. The problem in gx gy is I found our code was only taking into consideration one single CCA-U. So if PCRF sends more we are ignoring it. So to maintain it consistent with what we had on sessiond but have the way to send more than one credit on one single request, I had to do that. I guess n7 will suffer of the same issue due to the way sessiond reports the updates .